install/migrations/update_10.0.x_to_11.0.0/configs.php

```diff
@@ -44,10 +44,17 @@
     'set_followup_tech' => '0',
     'set_solution_tech' => '0',
     'is_demo_dashboards' => '0',
+    'planned_task_state' => '1',
 ]);
 $migration->addField('glpi_users', 'toast_location', 'string');
 
 $migration->addField('glpi_users', 'set_followup_tech', 'tinyint DEFAULT NULL');
 $migration->addField('glpi_users', 'set_solution_tech', 'tinyint DEFAULT NULL');
+$migration->addField(
+    'glpi_users',
+    'planned_task_state',
+    'int DEFAULT NULL',
+    ['update' => 1]
```

Comment:
In fact the value should remain `NULL` for existing users, to be sure that the default config is applied.
```suggestion
    'int DEFAULT NULL'
```